lib/matplotlib/type1font.py

```diff
@@ -139,6 +142,57 @@ def _split(self, data):
     _token_re = re.compile(br'/{0,2}[^]\0\t\r\v\n ()<>{}/%[]+')
     _instring_re = re.compile(br'[()\\]')
 
+    @staticmethod
+    def _decrypt(ciphertext, key, ndiscard=4):
+        """
+        Decrypt ciphertext using the Type-1 font algorithm
```

Comment:
Is there a reference to the "encryption" we can add here?  Also is decrypt the right word?  Its really just obfuscated at this point?

Reply:
The reference is the manual referenced in the file docstring. The manual uses the words "encrypt" and "decrypt", so I think it is best to follow its terminology.